new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodsShutdownOption.java

```diff
@@ -183,108 +288,117 @@ private static void createAndVerifyMiiDomain() {
-                .addEnvItem(new V1EnvVar()
-                    .name("SHUTDOWN_TYPE")
-                    .value("Graceful")))
+                    .value("-Djava.security.egd=file:/dev/./urandom ")))
             .adminServer(new AdminServer()
                 .serverStartState("RUNNING")
-                .serverPod(new ServerPod().shutdown(new Shutdown().shutdownType("Forced")))
-                .serverPod(new ServerPod().shutdown(new Shutdown().ignoreSessions(true)))
-                .serverPod(new ServerPod().shutdown(new Shutdown().timeoutSeconds(40L))))
+                .serverPod(new ServerPod()
+                    .shutdown(shutDownObject[1])))
             .addClustersItem(new Cluster()
                 .clusterName(clusterName)
                 .replicas(replicaCount)
                 .serverStartState("RUNNING")
-                .serverPod(new ServerPod().shutdown(new Shutdown().timeoutSeconds(80L)))
-                .serverPod(new ServerPod().shutdown(new Shutdown().ignoreSessions(true))))
+                .serverPod(new ServerPod()
+                    .shutdown(shutDownObject[2])))
             .configuration(new Configuration()
                 .model(new Model()
+                    .configMap(cmName)
                     .domainType(WLS_DOMAIN_TYPE)
                     .runtimeEncryptionSecret(encryptionSecretName)))
             .addManagedServersItem(new ManagedServer()
-                .serverName(managedServer1Name)
-                .serverPod(new ServerPod().shutdown(new Shutdown().timeoutSeconds(100L)))));
+                .serverStartState("RUNNING")
+                .serverStartPolicy("ALWAYS")
+                .serverName(indManagedServerName1)
+                .serverPod(new ServerPod()
+                    .shutdown(shutDownObject[3])))
+            .addManagedServersItem(new ManagedServer()
+                .serverStartState("RUNNING")
+                .serverStartPolicy("ALWAYS")
+                .serverName(indManagedServerName2)
+                .serverPod(new ServerPod()
+                    .shutdown(shutDownObject[4]))));
+    return domain;
+  }
 
+  // create domain resource and verify all the server pods are ready
+  private void createVerifyDomain(Domain domain) {
     // create model in image domain
     logger.info("Creating model in image domain {0} in namespace {1} using docker image {2}",
         domainUid, domainNamespace, miiImage);
     createDomainAndVerify(domain, domainNamespace);
 
-    // check that admin server pod exists in the domain namespace
-    logger.info("Checking that admin server pod {0} exists in namespace {1}",
+    // check that admin service exists in the domain namespace
+    logger.info("Checking that admin service {0} exists in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodExists(adminServerPodName, domainUid, domainNamespace);
+    checkServiceExists(adminServerPodName, domainNamespace);
```

Comment:
you can call checkPodReadyAndServiceExists(String podName, String domainUid, String namespace) method

Reply:
modified